Makefile

```diff
@@ -42,7 +42,8 @@ endif
 BUILD_X3=-X $(BUILD_INFO_IMPORT_PATH).BuildType=$(BUILD_TYPE)
 BUILD_INFO=-ldflags "${BUILD_X1} ${BUILD_X2} ${BUILD_X3}"
 
-SMART_AGENT_RELEASE=v5.9.1
+SMART_AGENT_VERSION_NUMBER=5.9.1
+SMART_AGENT_RELEASE=v$(SMART_AGENT_VERSION_NUMBER)
```

Comment:
seems unnecessary if only used once

Reply:
I removed SMART_AGENT_VERSION_NUMBER and used the "subst" to extract the version number.